docker-compose.yaml

```diff
@@ -82,10 +82,20 @@ services:
     networks:
     - ckan-multi
 
+  datapusher:
+    container_name: datapusher
```

Comment:
I'm not sure, but this should probably go into the override. Or if it stays here, it must be removed from there.